iothub/device/src/AmqpTransportSettings.cs

```diff
@@ -146,6 +146,12 @@ public TimeSpan OpenTimeout
             set => SetOpenTimeout(value);
         }
 
+        /// <summary>
+        /// A keep-alive for the transport layer in sending ping/pong control frames when using web sockets.
+        /// </summary>
+        /// <seealso href="https://docs.microsoft.com/dotnet/api/system.net.websockets.clientwebsocketoptions.keepaliveinterval"/>
```

Comment:
For consistency with the other protocol docs. 

```suggestion
        /// <remarks>
        /// This value is different from the protocol-level keep-alive packets that are sent over the overlaying AMQP transport protocol.
        /// </remarks>
        /// <seealso href="https://docs.microsoft.com/dotnet/api/system.net.websockets.clientwebsocketoptions.keepaliveinterval"/>
```